app/proc/processor_test.go

```diff
@@ -0,0 +1,28 @@
+package proc
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSetDefault(t *testing.T) {
+	p := Processor{
+		Conf: &Conf{},
+	}
+
+	assert.Equal(t, p.Conf.System.Concurrent, 0)
```

Comment:
I don't think it makes sense to test zero values of empty structure.

Reply:
I agree